docs: add additional payments documentation (#27379)
MystenLabs/sui: docs/content/onchain-finance/agentic-payments/index.mdx

  spending policies, sponsored gas, settlement verification, and production
  safety patterns.
keywords:
  - agentic payments
  - autonomous agents
  - agent wallet
  - spending mandate
  - settlement verification
  - production hardening
  - agent gas sponsorship
goal:
  description: >-
    Reader can navigate the agentic payments documentation to find the right
    guide for their use case
  requires:
    - has_frontmatter:
        - title
        - description
        - keywords
      label: Has required frontmatter fields
    - has_questions: true
      label: Needs questions for AI search visibility
    - has_answer: true
      label: Needs answer summary for AI citation
builder_paths:
  - path_id: agentic-payments
    path_name: Agentic Payments
    step: Overview
    stage: Overview
questions:
  - What are agentic payments on Sui?
  - How do I build an autonomous payment agent on Sui?
  - What documentation covers agentic payment patterns?
answer: >-
  Agentic payments on Sui let autonomous backend services sign and submit
  payment transactions without human interaction, using spending policies,
  sponsored gas, and settlement verification for safe, bounded operation.
---

> Agentic payments let autonomous backend services sign and submit payment transactions on Sui without human interaction. An agent holds its own key pair, operates within spending policies set by the owner, and uses sponsored gas so it never needs to hold SUI for fees.

Author: ```suggestion
Agentic payments let autonomous backend services sign and submit payment transactions on Sui without human interaction. An agent holds its own key pair, operates within spending policies set by the owner, and, in some instances, use sponsored gas so it doesn't need to hold SUI for fees.
```

Building a payment agent requires four capabilities:

1. **A wallet the agent can use.** The agent needs a key pair it can access programmatically, stored securely and rotatable without downtime.

> 2. **Spending boundaries.** The agent's owner defines what the agent can spend, to whom, and for how long. These boundaries are enforced onchain so the agent cannot exceed them even if compromised.

Author: ```suggestion
2. **Spending boundaries.** The agent's owner defines what the agent can spend, to whom, and for how long. These boundaries are enforced onchain so the agent cannot exceed them.
```

4. **Production safety.** Kill-switches, idempotent retries, and circuit breakers prevent runaway spending and ensure graceful recovery from failures.

import DocCardList from '@theme/DocCardList';

<DocCardList items={[
{
type: 'link',
label: 'Spending Policies and Mandates',
href: '/onchain-finance/agentic-payments/spending-policies',
description: 'Enforce onchain spending caps, allowlists, expiry, and revocation.',
},
{
type: 'link',
label: 'Recurring and Streaming Payments',
href: '/onchain-finance/agentic-payments/recurring-payments',
description: 'Scheduled agent payments and onchain streaming with linear unlock.',
},
{
type: 'link',
label: 'Settlement Verification',
href: '/onchain-finance/agentic-payments/settlement-verification',
description: 'Verify payment finality before releasing goods or services.',
},
{
type: 'link',
label: 'Production Hardening',
href: '/onchain-finance/agentic-payments/production-hardening',
description: 'Kill-switches, retry safety, idempotency, monitoring, and Mainnet checklist.',
},
]} />

> 

Author: ```suggestion
```
